services/src/main/java/org/exoplatform/news/NewsServiceImpl.java

```diff
@@ -1198,6 +1198,9 @@ public boolean canEditNews(News news, String authenticatedUser) {
     if (spaceService.isManager(currentSpace, authenticatedUser)) {
       return true;
     }
+    if (spaceService.isRedactor(currentSpace, authenticatedUser) && news.isDraftVisible()) {
```

Comment:
I think that this condition should be verified for a posted article draft which was draft visible before posting, normally the redactors in that case should be able to edit the article